src/index.html

```diff
@@ -17,6 +29,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
```

Comment:
Consider making the `alt` attribute more descriptive to improve accessibility. Instead of 'moyo_logo', you could use something like 'Moyo Company Logo'.